scss/tech.scss

```diff
@@ -15,18 +15,29 @@
   margin: 10px 20px;
   min-height: 280px;
   width: 180px;
-
+  
   &::before {
+    transition: top 0.5s ease, left 0.5s ease;
     bottom: 10px;
     border: 1px solid $green;
     border-radius: 5px;
     content: '';
+    height: 100%;
     left: -10px;
     right: 10px;
     position: absolute;
     top: -10px;
+    width: 180px;
     z-index: -1;
   }
+
+  &:hover {
+    &::before {
+      left: 10px;
+      top: 10px;
+      transition: top 0.2s ease, left 0.2s ease;
```

Comment:
This line is not needed. Try deleting it and you will find out that is not necessary 👍